Source/Tests/Unit Tests/FPGA Register IO Testing/FPGA Register IO Tests/targets.ini

```diff
@@ -0,0 +1,5 @@
+[Supported Targets]
```

Comment:
Please add all available targets, not only PharLap.

Reply:
These are only unit tests which are require an FPGA target. The Controller for these tests are not important. Running it twice will only add to the overal runtime of the ats with no benefit at all.

Reply:
I left PharLap as it's the first OS which gets installed on the ATS machine.